Sources/Core/Microsoft.StreamProcessing/Ingress/SubscriptionBase.cs

```diff
@@ -160,6 +161,16 @@ public abstract class DisorderedSubscriptionBase<TIngressStructure, TPayload, TR
         [DataMember]
         protected long currentTime = StreamEvent.MinSyncTime;
 
+#if DEBUG
+        /// <summary>
+        /// Currently for internal use only - do not use directly.
+        /// Used for debug purposes only - very close to currentTime, but is only updated due to real events/punctuations/low watermarks processed, and not the reorder buffer latency
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [DataMember]
+        protected long lastEventTime = StreamEvent.MinSyncTime;
```

Comment:
Instead of a field and a getter method, can we make this a property with a protected get and private or protected set?

Reply:
I think this would over complicate the T4 (e.g., the partitioned case can't be a simple getter property), especially considering this is debug only code

---
In reply to: [320955088](https://github.com/microsoft/Trill/pull/108#discussion_r320955088) [](ancestors = 320955088)